FULL_HELP_DOCS.md

```diff
@@ -424,6 +424,7 @@ Initialize a Soroban project with an example contract
 * `-f`, `--frontend-template <FRONTEND_TEMPLATE>` — An optional flag to pass in a url for a frontend template repository.
 
   Default value: ``
+* `-o`, `--overwrite` — Overwrite all existing files.
```

Comment:
I don't think we should add the short option `-o`, opened a PR removing it and why is mentioned there:
- https://github.com/stellar/stellar-cli/pull/1483